src/compute-client/src/controller.rs

```diff
@@ -292,6 +292,25 @@ impl<T: ComputeControllerTimestamp> ComputeController<T> {
         self.arrangement_exert_proportionality = value;
     }
 
+    /// Returns `true` iff all collections on all clusters have been hydrated.
+    pub fn check_clusters_hydrated(&self) -> bool {
```

Comment:
Nit: The `check_` prefix indicates (to me) that this will return a `Result` or panic on error, while this method just implements a predicate. I'd call it `clusters_hydrated` instead (same for `check_any_replica_hydrated` below).